src/style.css

```
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap");
+
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__image {
+  padding-top: 32px;
+  padding-bottom: 40px;
```

```suggestion
  margin-bottom: 40px;
```
Используй короткую запись, и верхний отступ укажи как padding у самого container